### DISCUSSION TITLE
Error in Build scripts "Embed Pods Frameworks"

### DISCUSSION THREAD

**Discussion by noorbhatia** [codemagic-ci-cd/codemagic-docs#2499] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

build ipa

### Expected results

Expected to build

### Actual results

getting this error 
```
PhaseScriptExecution [CP]\ Embed\ Pods\ Frameworks /Users/builder/Library/Developer/Xcode/DerivedData/Hunbun-bqyendvkgvmdunffgobosswxpind/Build/Intermediates.noindex/ArchiveIntermediates/Hunbun\ Dev/IntermediateBuildFilesPath/Hunbun.build/Debug-iphoneos/Hunbun.build/Script-EF4F21BAA91994FD9F4C7D52.sh (in target 'Hunbun' from project 'Hunbun')
```

### Build id (optional)

662f42dc5e39bd0ba033f719

**Comment by dtrdic** [comment]
Hello @noorbhatia, the error message indicates that the build process failed during the phase where CocoaPods frameworks are being embedded into your project. This could occur for various reasons..
- Try to use the latest cocoapods 1.15.2 version
- try to clean you project locally by running this commands;
```
cd ios
pod cache clean --all
rm -rf ~/Library/Caches/CocoaPods
rm -rf Pods
rm -rf ~/Library/Developer/Xcode/DerivedData/*
pod deintegrate
flutter clean
pod install
```
- try also to remove your podfile.lock file
- add the archive flag to your build command
```
xcode-project build-ipa \
  --workspace "$CM_BUILD_DIR/Hunbun/$XCODE_WORKSPACE" \
  --scheme "$XCODE_SCHEME" \
  --archive-flags="-destination 'generic/platform=iOS'"   
```

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
